users/views.py

```diff
@@ -1,3 +1,21 @@
 from django.shortcuts import render
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from users.serializers import RegistrationSerializer
 
 # Create your views here.
+
+@api_view(['POST'])
+def registrationview(request):
+
+    serializer = RegistrationSerializer(data = request.data)
+    data = {}
+    if serializer.is_valid():
+         user = serializer.save()
+         data['success'] = True
+         data['message'] = "Registration Successful"
+    else:
+        data = serializer.errors
```

Comment:
We expect custom error messages friendly enough to be displayed in the frontend app/website.

Reply:
Could you please have a look at my serializers.py file and let me know if these were the sort of error messages that were expected ?

Reply:
Sure will check